malwarebytes.sls

```diff
@@ -6,12 +6,25 @@
 {% endif %}
 # Source: http://www.malwarebytes.org
 malwarebytes:
-  '3.5.1.2522':
-    full_name: 'Malwarebytes Anti-Malware version 2.0.4.1028'
-    installer: 'https://data-cdn.mbamupdates.com/web/mb3-setup-consumer/mb3-setup-consumer-3.5.1.2522-1.0.374-1.0.5597.exe'
+#  {% for version in ['4.3.0.210'] %}
+#  '{{ version }}':
+#    full_name: 'Malwarebytes Anti-Malware version {{ version }}'
+#    installer: 'https://data-cdn.mbamupdates.com/web/mb4-setup-consumer/MBSetup.exe'
+#    install_flags: '/SP- /VERYSILENT /SUPPRESSMSGBOXES /NORESTART'
+#    uninstaller: '{{ PROGRAM_FILES }}\Malwarebytes Anti-Malware\unins000.exe'
+#    uninstall_flags: '/SP- /VERYSILENT /SUPPRESSMSGBOXES /NORESTART'
+#    msiexec: False
+#    locale: en_US
+#    reboot: False
+#  {% endfor %}
+#
+  '4.3.0.210':
```

Comment:
Looks like they've migrated to a "latest" paradigm as well, no longer offering older versions. May need to use `latest` as the version as we do with chrome.

I still found a url for 3.8 though: https://data-cdn.mbamupdates.com/web/mb3-setup-consumer/mb3-setup-consumer-3.8.3.2965-1.0.629-1.0.13137.exe

Reply:
@twangboy but how do I deal with 'latest' IF the version umber is part of the 'full_name' that win pkg tries to match against?

Reply:
I don't know... perhaps it won't work in this case then.